package.json

```diff
@@ -45,6 +45,7 @@
     "standard": "10.0.2",
     "uglify-js": "github:mishoo/UglifyJS2#harmony"
   },
+  "yarnDependency": "0.23.3",
```

Comment:
Could we put this in a 'cli-engine' section to denote it's a config specific to this project?